pydm/tests/widgets/test_label.py

```diff
@@ -540,8 +540,10 @@ def test_value_changed_incorrect_display_format(qtbot, signals, capfd, value, di
     signals.new_value_signal[type(value)].emit(value)
     pydm_label.displayFormat = display_format
 
-    out, err = capfd.readouterr()
-    assert expected in err
+    # Make sure logging capture the error, and have the correct error message
+    for record in caplog.records:
+        assert record.levelname == 'ERROR'
```

Comment:
Wouldn't it be better to switch to `levelno` and compare with `logging.ERROR` instead?